### ISSUE TITLE
Why built my ardiuino your examples not with V1.4.0 but with V1.3.1 and how should work the call backs?

### ISSUE THREAD

**Issue by sosojaja** [LennartHennigs/ESPTelnet#30]
Hi,
my main problem war, that I could not enter something in the Telnet connection.
I could enter things in the serial connection ant it will be seen in the Telnet connection but nothing happend. :-/
I thing I do something wrong or nobody use this function till now.

But now I have the problem that all examples in v1.4.0 didn't run anymore with V1.4.0.
Like I think it will be missing a library but I don't know which.
Could you tell me how could I see which libraries ESP Telnet are needed?

I tried it with arduino V1.8.9 and V2.0.2 but at each I could not built a running file! :-(

Thank you very much ahead for your help and I hope this questins are not to low performant, I'm didn't work very much mit this arduino things tillnow.

**Comment by LennartHennigs** [comment]
Hey, 
thanks for pointing this out. 
There was a bug in 1.4.0.
Please use either 1.3.1 or the newly released 2.0.0 which fixes this and adds new functionality.

Cheers
l.

**Comment by LennartHennigs** [comment]
**P.S.** If you want to use version 1.4.0 change line 58 in `ESPTelnet.h` from...

```
    TCPServer server;
```

...to...

```
    TCPServer server =  TCPServer(23);
```
